doctr/travis.py

```diff
@@ -146,32 +146,20 @@ def setup_GitHub_push(deploy_repo, auth_type='deploy_key', full_key_path='github
     For ``auth_type='deploy_key'``, this sets up the remote with ssh access.
     """
 
-    if branch_whitelist is None:
+    if not branch_whitelist:
         branch_whitelist={'master'}
 
     if require_master is not None:
```

Comment:
I checked on GitHub and there are no `import doctr`s anywhere. I was thinking of using the Python API for a project, but I haven't implemented it yet. We shouldn't be too worried about changing it I'd say.